core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -4717,6 +4719,81 @@ object Stream extends StreamLowPriority {
       def combine(x: Stream[F, O], y: Stream[F, O]) = x ++ y
     }
 
+  /** `Align` instance for `Stream`.
+    * * @example {{{
+    * scala> import cats.implicits._
+    * scala> Stream(1,2,3).align(Stream("A","B","C","D","E")).toList
+    * res0: List[cats.data.Ior[Int,String]] = List(Both(1,A), Both(2,B), Both(3,C), Right(D), Right(E))
+    * }}}
+    *
+    */
+  implicit def alignInstance[F[_]](implicit F: Functor[Stream[F, ?]]): Align[Stream[F, ?]] =
```

Comment:
Can't check now, but doesn't Stream have a Functor "just like that", without any parameters?